From ef466d982a1b3c6917938be6d426acd6b40de4b9 Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Sun, 19 Nov 2017 15:06:28 +0000
Subject: [PATCH 1/1] Set player passthrough as the default player input method

* This supports gnome-mpv, mpv and vlc without the user
  needing to change the player-input method settings.

Signed-off-by: Joey Riches <josephriches@gmail.com>
---
 src/app/models/localstorage/Settings/streaming.js | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/app/models/localstorage/Settings/streaming.js b/src/app/models/localstorage/Settings/streaming.js
index 57681849..6b1ca59b 100644
--- a/src/app/models/localstorage/Settings/streaming.js
+++ b/src/app/models/localstorage/Settings/streaming.js
@@ -35,7 +35,7 @@ export default Fragment.extend({
 	players: fragment( "settingsStreamingPlayers", { defaultValue: {} } ),
 
 	oauth: attr( "boolean", { defaultValue: true } ),
-	player_input: attr( "string", { defaultValue: ATTR_STREAMING_PLAYER_INPUT_STDIN } ),
+	player_input: attr( "string", { defaultValue: ATTR_STREAMING_PLAYER_INPUT_PASSTHROUGH } ),
 	player_no_close: attr( "boolean", { defaultValue: false } ),
 	hls_live_edge: attr( "number", { defaultValue: 3, min: 1, max: 10 } ),
 	hls_segment_threads: attr( "number", { defaultValue: 1, min: 1, max: 10 } ),
-- 
2.15.0

